coding-guides/starboard.md

```diff
@@ -0,0 +1,199 @@
+# Starboard
+
+This is a long awaited feature, requested by many people.
+
+Let's begin by talking about what a starboard is. This is an example taken from the Discord.js Official Server.
+
+![Starboard](/assets/starboard.png)
+
+> Before we start, there is one thing you need to now. This code is compatible with only Guidebot Class. This is due to a restriction with passing `client` to the messageReactionAdd and messageReactionRemove events.
+
+So, let's begin!
```

Comment:
I'm not really fond of jumping straight into a codeblock, maybe try explaining what you're trying to accomplish first.

Reply:
You explain what you're doing in the block in a sort of past tense, you're not really explaining what you're attempting to do.

For example, once you've explained what a starboard is (above) go like...

_"To make a functional starboard, you'll need to monitor for certain reactions, when a user reacts..."_ and go from there.